scripts/managed-gateway-control.py

```
@@ -1601,8 +1603,8 @@ def _validate_runtime_environment(script: str, environment: dict[str, str]) -> N
     module = importlib.util.module_from_spec(spec)
     try:
         spec.loader.exec_module(module)
-        validator = getattr(module, "validate_runtime_env")
-        result = validator(environment)
+        validator = getattr(module, "validate_managed_gateway_env")
```

Ruff (0.16.3)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)